build.gradle.kts

```diff
@@ -0,0 +1,30 @@
+plugins {
+	java
+	id("org.springframework.boot") version "3.3.4"
+	id("io.spring.dependency-management") version "1.1.6"
+}
+
+group = "com.softdevsix"
+version = "v1"
```

Comment:
According to [semantic versioning ](https://semver.org/) It would be better to leave it as 0.0.1

Reply:
done